api/models/task.go

```diff
@@ -23,7 +23,7 @@ const (
 
 const (
 	// FormatDefault ...
-	FormatDefault = ""
+	FormatDefault = "default"
```

Comment:
I am not so sure: check https://github.com/iron-io/functions/blob/master/api/runner/protocol/factory.go#L33